steam_wrapper/steam_wrapper.py

```diff
@@ -55,13 +55,17 @@ class Message(t.NamedTuple):
     text: str
     always_show: bool
 
-    def show(self) -> bool:
-        logging.warning(self.title)
+    def show(self, subst: t.Dict[str, str] = None) -> bool:
```

Comment:
**kwargs would also work here, I guess.

Reply:
I thought about that, but using a single variable feels more future-proof. In case we'll need another arg, like `force_always_show` or something.